### ISSUE TITLE
[Keyboard Navigation-Makecode-Documentation]Various Buttons like Edit,Download, Run are not accessible using keyboard

### ISSUE THREAD

**Issue by v-mesala** [Microsoft/pxt#2798]
**User Experience:**
User who depends on Keyboard with Screen Reader may get restricted from accessing all the functionalities if that element is not accessible using keyboard.

**Test Environment**
OS: RS2 Version 1703(OS Build 15063.2)
Platform: Edge

**Repro Steps**
1-Navigate to url:-https://makecode.microbit.org/reference/basic/show-number
2-Navigate to all the elements under show number page
3-Verify that some buttons are accessible or not

**Actual Result**
Buttons like 'edit' ,'download' 'run' are not accessible using keyboard

**Expected Result**
Buttons like 'edit' ,'download' 'run' must be accessible using keyboard

**MAS Reference**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}

**Refer Attachment:**
[MAS 2.1.1_NotAccesibleUsingKeyboard.zip](https://github.com/Microsoft/pxt/files/1248570/MAS.2.1.1_NotAccesibleUsingKeyboard.zip)

**Comment by veler** [comment]
This issue is duplicated with a closed one [https://github.com/Microsoft/pxt/issues/2529](https://github.com/Microsoft/pxt/issues/2529). I tried to reproduce it on my machine, and everything works fine as expected and described by the current issue.

**Comment by veler** [comment]
Fix available on [https://makecode.microbit.org/](https://makecode.microbit.org/)

**Comment by v-mesala** [comment]
Verified on URL: https://makecode.microbit.org/reference/basic/show-number and issue is fixed now. Hence closing the bug
